bitrise.yml

```diff
@@ -0,0 +1,49 @@
+---
+format_version: '6'
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+project_type: ios
+trigger_map:
+- push_branch: "*"
+  workflow: primary
+- pull_request_source_branch: "*"
+  workflow: primary
+workflows:
+  primary:
+    steps:
+    - activate-ssh-key@4.0.3:
+        run_if: '{{getenv "SSH_RSA_PRIVATE_KEY" | ne ""}}'
+    - git-clone@4.0.12: {}
+    - cache-pull@2.0.1: {}
+    - script@1.1.5:
+        title: Do anything with Script step
+    - certificate-and-profile-installer@1.10.1: {}
+    - carthage@3.1.5:
+        inputs:
+        - github_access_token: "$CARTHAGE_GITHUB_ACCESS_TOKEN"
+        - carthage_options: "--platform iOS --cache-builds"
+        - carthage_command: bootstrap
+    - xcode-test@1.18.17:
+        deps:
+          brew:
+          - name: swiftlint
+        inputs:
+        - project_path: "$BITRISE_PROJECT_PATH"
+        - scheme: "$BITRISE_SCHEME"
+    - xcode-test@1.18.17:
+        inputs:
+        - xcpretty_test_options: --color --report html --output "${BITRISE_DEPLOY_DIR}/xcode-atb-test-results-${BITRISE_SCHEME}.html"
+        - scheme: AtbIntegrationTests
```

Comment:
I'm not a BitRise expert, just a thought: since we know that this one may fail from time to time (like, for instance [here](https://app.bitrise.io/build/5c8f91f4fc8467f2) ) then, maybe we could set 'ShouldRetryTestOnFail' to 'yes' for ATB and see whether retry help somewhat?

Reply:
Great idea, I was planning to fix them as part of LHF so that works well.

Reply:
That's done!